funcx/executors/high_throughput/worker_map.py

```diff
@@ -11,26 +11,29 @@ class WorkerMap(object):
     def __init__(self, worker_count):
         logger.info("Start")
         self.worker_count = worker_count
-        self.worker_counts = {'slots' : self.worker_count}
+        self.worker_counts = {'slots': self.worker_count, 'RAW': 0}
+        self.ready_worker_counts = {}
```

Comment:
Maybe `self.ready_workers` is a better name here?
This is a bit confusing with the function `ready_worker_count` later.
But feel free to ignore if I am not understanding correctly.

Reply:
This is a great point. Updating accordingly.  Renaming everything to be descriptive :eyes:

Reply:
I changed them to:
ready_worker_type_counts
total_worker_type_counts

Seems more readable to me.